bundles/core/org.eclipse.smarthome.core.persistence/src/main/java/org/eclipse/smarthome/core/persistence/ModifiablePersistenceService.java

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
could you re-run "mvn license:format" to update the headers to the latest version?